src/test/java/org/jitsi/meet/test/base/AbstractBaseTest.java

```diff
@@ -242,7 +242,7 @@ public JitsiMeetUrl getJitsiMeetUrl()
         String token = System.getProperty(TOKEN_PNAME);
 
         JitsiMeetUrl u = participants.getJitsiMeetUrl().setRoomName(currentRoomName);
-        if (token.length() > 0)
+        if (token != null)
         {
```

Comment:
Or you want to check for both which is done with `StringUtils.isBlank` or `StringUtils.isNotBlank`